src/azure-cli/azure/cli/command_modules/appconfig/_help.py

```diff
@@ -92,14 +92,16 @@
 type: command
 short-summary: List key-values.
 examples:
-  - name: List all key-values.
-    text: az appconfig kv list -n MyAppConfiguration
+  - name: List all key-values with null label.
+    text: az appconfig kv list -n MyAppConfiguration --label \\0
```

Comment:
If you want to list all key-values with null label, why not just don't specify --label as before or you can give --label a default value to represent null label flag?

Reply:
Besides "*" we also support "," as key/label filter. For example, if I want to list items with two labels, I can do --label "labelA,labelB". For null label items, we used to use empty string to represent it. But it breaks if we want to list items with labelA and null label.  --label ",labelA" will be treated as just labelA in our service backend. The reason for using "\0" is that it is the escape for null in python. There is a principle that we are trying to follow. CLI won't interpret inputs for key/label filters. Thus, by using "\0" CLI will just blindly get the input arguments, unescape, url encode and send out request. When backend service gets the value, it will map to null and labelA. 

---
In reply to: [347081003](https://github.com/Azure/azure-cli/pull/11036#discussion_r347081003) [](ancestors = 347081003)